app/code/Magento/Store/Url/Plugin/RouteParamsResolver.php

```diff
@@ -63,6 +63,7 @@ public function beforeSetRouteParams(
     ) {
         if (isset($data['_scope'])) {
             $subject->setScope($data['_scope']);
+            $subject->setRouteParam('_scope',$data['_scope']);
```

Comment:
There should be a space here according to PSR-2 but I'm not sure if Travis is not broken currently.

Reply:
ok